orchagent/vnetorch.cpp

```diff
@@ -163,6 +163,47 @@ bool VNetVrfObject::addRoute(IpPrefix& ipPrefix, tunnelEndpoint& endp)
     return true;
 }
 
+void VNetVrfObject::increaseNextHopRefCount(const nextHop& nh)
+{
+    /* Return when there is no next hop (dropped) */
+    if (nh.ips.getSize() == 0)
+    {
+        return;
+    }
+    else if (nh.ips.getSize() == 1)
+    {
+        NextHopKey nexthop(nh.ips.to_string(), nh.ifname);
+        if (nexthop.ip_address.isZero())
```

Comment:
Minor comment - can you provide open and closing braces here and the below "if" condition?